android/src/main/java/com/swmansion/rnscreens/ScreenViewManager.kt

```diff
@@ -35,6 +37,18 @@ class ScreenViewManager : ViewGroupManager<Screen>(), RNSScreenManagerInterface<
         setActivityState(view, activityState.toInt())
     }
 
+    override fun updateState(
+        view: Screen,
+        props: ReactStylesDiffMap?,
+        stateWrapper: StateWrapper?
+    ): Any? {
+        if (BuildConfig.IS_NEW_ARCHITECTURE_ENABLED) {
+            view.fabricViewStateManager.setStateWrapper(stateWrapper)
+            return null
```

Comment:
Why we're returning `null` here instead of returning `super.updateState` like below? Is it breaking some logic of the screens there?